searx/templates/simple/elements/engines_msg.html

```diff
@@ -1,27 +1,34 @@
 <div id="engines_msg">
   {% if not results and not answers %}
   <details class="sidebar-collapsable" open>
+    <summary class="title" id="engines_msg-title">{{ _('Messages from the search engines') }}</summary>
   {% else %}
   <details class="sidebar-collapsable">
+    <summary class="title" id="engines_msg-title">{{ _('Response time') }}: {{ timings[-1].total | round(1) }} {{ _('seconds') }}</summary>
   {% endif %}
-    <summary class="title" id="engines_msg-title">{{ _('Messages from the search engines') }}</summary>
-    <div class="dialog-error" role="alert">
-      {{ icon_big('warning') }}
-      <div>
-        <p>
-          <strong>{{ _('Error!') }}</strong>
-          {{ _('Engines cannot retrieve results') }}:
-        </p>
-        {%- for engine_name, error_type in unresponsive_engines -%}
-          <p>{{- engine_name }} (
-            <a href="{{ url_for('stats', engine=engine_name|e) }}"
-               title="{{ _('View error logs and submit a bug report') }}">
-               {{- error_type -}}
-            </a>
-            ){{- '' -}}
-          </p>
-        {%- endfor -%}
-      </div>
-    </div>
+    <table class="engine-stats" id="engines_msg-table">
+      {%- for engine_name, error_type in unresponsive_engines -%}
+      <tr>
+        <td class="engine-name">
+          <a href="{{ url_for('stats', engine=engine_name|e) }}"
+             title="{{ _('View error logs and submit a bug report') }}">
+             {{- engine_name -}}
+          </a>
+        </td>
+        <td class="response-error">{{- error_type -}}</td>
+      </tr>
+      {%- endfor -%}
+      {%- for timing in timings | reverse -%}
+      <tr>
+        <td class="engine-name"><a href="{{ url_for('stats', engine=timing.engine|e) }}">{{ timing.engine }}</a></td>
+        <td class="response-time">
+          <div class="bar-chart-value">{{- timing.total | round(1) -}}</div>
+          <div class="bar-chart-graph" aria-labelledby="{{timing.engine}}_time" aria-hidden="true">
+            <div class="bar-chart-bar bar{{ (100 * timing.total / timings[-1].total) | round | int }}"></div>
```

Comment:
With how its written, the UI is now some business logic to calculate the bar-chart length. For this iteration, I think this approach is simplest, as doing it in the backend will involve mapping the `request.timings` to a new array object with this calculation. I also don't suspect we will be changing this anytime to worry about it too much.